DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.WebAPI/Controllers/ActivityController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Activity.External.WebAPI.Services;
+using GitHubExtension.Security.WebApi.Library.Exceptions;
+using Microsoft.AspNet.Identity;
+using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.Activity.External.WebAPI.Controllers
+{
+    [Authorize]
+    public class ActivityController : ApiController
+    {
+        private IGitHubService _gitHubService;
+
+        public ActivityController(IGitHubService gitHubService)
+        {
+            _gitHubService = gitHubService;
+        }
+
+        [Route(RouteConstants.GetGitHubActivityRoute)]
+        public async Task<IHttpActionResult> GetGitHubActivity([FromUri] int page)
+        {
+            var claimsIdentity = User.Identity as ClaimsIdentity;
+
+            string fullRepositoryName = claimsIdentity.FindFirstValue("CurrentProjectName");
```

Comment:
I think it is possible to move part of claims logic to owin middleware(access token) for current project I'll write en extension method. Understood